src/frontend/src/lib/canisters/backend.canister.ts

```diff
@@ -1,15 +1,21 @@
 import type {
 	_SERVICE as BackendService,
 	CustomToken,
+	PendingTransaction,
+	SelectedUtxosFeeResponse,
 	UserProfile,
 	UserToken
 } from '$declarations/backend/backend.did';
 import { idlFactory as idlCertifiedFactoryBackend } from '$declarations/backend/backend.factory.certified.did';
 import { idlFactory as idlFactoryBackend } from '$declarations/backend/backend.factory.did';
 import { getAgent } from '$lib/actors/agents.ic';
+import { backendCanisterError } from '$lib/canisters/backend.errors';
```

Comment:
There was already a PR to introduce this `backendCanisterError`?

Reply:
It's a good idea, just not sure if we want to have a single function for all error. In ic-js we have an error parser per function.

Reply:
It was previously done for Signer. And in a parallel PR (https://github.com/dfinity/oisy-wallet/pull/2629), I created a similar function for Backend. 

Re. the structure itself - I also have a feeling that having an error parser per function will be more usable in the future. I will adjust it for both BE and signer in the above mentioned PR. Let's review and merge that one first and then come back to this one.

Reply:
Super. Ping me when you are ready to review again. I'll also need this new pattern to add a function to the backend api.

Reply:
@peterpeterparker https://github.com/dfinity/oisy-wallet/pull/2629 is ready for review. I will update this one after we merge the errors parsers PR.

Reply:
@peterpeterparker aaand this PR is now also ready for re-review.